packages/components/src/buttons/src/EmbeddedButton.tsx

```diff
@@ -26,6 +26,23 @@ import styles from "./EmbeddedButton.module.css";
 
 export const GlobalEmbeddedButtonCssSelector = "hop-EmbeddedButton";
 
+// TODO: Remove this when Orbiter is not used anymore
+type OrbiterVariants = "informative" | "warning";
+
+export function mapOrbiterToHopperVariants(variant: EmbeddedButtonVariants): EmbeddedButtonVariants {
```

Comment:
although there are different values for the Button variants, I didn't actually see these used in Orbiter's useSlots function. Wouldn't this mean that we don't actually need to do the mapping? Or is it possible that someone used it as a custom slot somewhere?
I kept going back and forth on this..

Reply:
my bad, I see now that it's because this was used in Tag